scripts/Synapse/generateIRISCaseFiles.ipynb

```diff
@@ -5,25 +5,16 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "### extractTSV\n",
-        "This is the final step after appropriately updating the ECR datastore (`person_id`,`iris_id`,and`incident_id`). \n",
+        "### generateIRISCaseFiles\n",
+        "This is the final step after appropriately updating the ECR datastore (person_id,iris_id,andincident_id).\n",
```

Comment:
Do we need to make the same corrections here with the ` and should there be a space between 'and' and 'incident_id'?

Reply:
Good catch!